app/assets/javascripts/components/client/ClientProfile.jsx

```diff
@@ -75,70 +80,74 @@ class ClientProfile extends React.Component {
         <ClientPageHeader client={this.props.client} page={"profile"} />
 
         <div className="container">
-          <div className="card-bg">
-            <h3>Personal Details</h3>
-            <h4>Name</h4>
-            {client.first_name} {client.last_name}
-            
-            <h4>Education</h4>
-            {client.education}
-
-            <h4>Income</h4>
-            {client.client_income}
-
-            <h4>Family Income</h4>
-            {client.family_income}
-
-            <h4>Help</h4>
-            {client.help}
-
-            <h4>Flee Country</h4>
-            {_flee_country}
-
-            <h4>Citizen Spouse</h4>
-            {_citizen_spouse}
-
-            <h4>Citizen Child</h4>
-            {_citizen_child}
-
-            <h4>Victim Crime</h4>
-            {client.victim_crime}
-
-            <h4>Living With Parents</h4>
-            {_living_w_parents}
-
-            <h4>Initial Intake</h4>
-            {client.initial_intake}
-
-            <h3>Contact Information</h3>
-            <h4>Phone Number</h4>
-            {client.phone_number}
-            <h4>Country</h4>
-            {client.country}
-            <h4>State</h4>
-            {client.state}
-            <h4>Postal Code</h4>
-            {client.postal_code}
-            <h4>City</h4>
-            {client.city}
-            <h4>Street</h4>
-            {client.street}
-
-            <h3>Immigration History</h3>
-
-            <h4>Court Date</h4>
+          <div className="clients-page-main-container client-profile card-bg">
+            <div className="profile-section">
+              <h2 className="title">Personal Details</h2>
+              <h3>Name</h3>
```

Comment:
Do you think we should create a separate component that takes in a label, and text, and then formats it?

Reply:
I'm indifferent, since the html is minimal. @amyshan @lbkchen what are your thoughts?